graylog2-web-interface/src/components/streams/StreamList.jsx

```diff
@@ -59,20 +57,12 @@ const StreamList = createReactClass({
   },
 
   render() {
-    if (this.props.streams.length > 0) {
```

Comment:
It looks like we are now always displaying the `EmptyEntity` component for the streams list.